modules/virt-vm-behavior-dr.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-disaster-recovery.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-vm-behavior-dr_{context}"]
+= VM behavior during disaster recovery scenarios
+
+VMs typically act similarly to pod-based workloads during failover and relocate disaster recovery flows.
+
+[discrete]
+[id="dr-failover_{context}"]
+== Failover
+
+Use failover when there is a critical failure in the primary environment that makes it impractical or impossible to use relocation to move the workload to a secondary environment. When failover is executed, the storage is fenced from the primary environment, the I/O to the VM disks is abruptly halted, and the VM restarts in the secondary environment using the replicated data.
+
+You should expect some amount of data loss due to failover. The extent of loss depends on whether you use Metro-DR, which uses synchronous replication, or Regional-DR, which uses asynchronous replication. Because Regional-DR uses snapshot-based replication intervals, the window of data loss is proportional to the replication interval length. When the VM is restarted, the operating system may need to perform crash recovery.
```

Comment:
**suggestions**:
- We can remove `some amount of`.
- `When the VM restarts, the operating system might perform crash recovery.`